index.html

```diff
@@ -834,13 +859,30 @@ <h2 id="state-transitions" class="informative">
           </tr>
           <tr>
             <td>
+              <dfn>[[\serializedModifierData]]</dfn>
+            </td>
+            <td>
+              A list containing the serialized string form of each <a data-lt=
+              "PaymentDetailsModifier.data">data</a> member for each
+              corresponding item in the sequence
+              <a>[[\details]]</a>.<a data-lt="PaymentDetails">modifier</a>, or
+              null if no such member was present.
+            </td>
+          </tr>
+          <tr>
+            <td>
               <dfn>[[\details]]</dfn>
             </td>
             <td>
               The current <a>PaymentDetails</a> for the payment request
               initially supplied to the constructor and then updated with calls
               to <a data-lt=
-              "PaymentRequestUpdateEvent.updateWith">updateWith()</a>.
+              "PaymentRequestUpdateEvent.updateWith">updateWith()</a>. Note
+              that all <a data-lt="PaymentDetailsModifier.data">data</a>
+              members of <a>PaymentDetailsModifier</a> instances contained in
+              the <a data-lt="PaymentDetails.modifiers">modifiers</a> member
+              will be removed, as they are instead stored in serialized form in
+              the [[\serializedModifierData]] internal slot.
```

Comment:
nit: link to `<a>[[\serializedModifierData]]</a>`. I'll eventually teach ReSpec about internal slots :(